cargo-public-api/src/main.rs

```diff
@@ -365,21 +365,21 @@ fn collect_public_api_from_commit(
     } else {
         None
     };
-    let mut build_options = BuildOptions::default()
+    let mut builder = Builder::default()
```

Comment:
Wouldn't it be better to make it clear what kind of builder this is directly at this first-use site?
```suggestion
    let mut builder = rustdoc_json::Builder::default()
```
and same thing for the other use-sites below.